### PR TITLE
Scrape BI function descriptions and add syntax highlighting support

### PR DESCRIPTION
**Pull request by Cuel**
Running the script now will scrape all BI functions and try to get their descriptions

Some BI functions descriptions are a mess and hard to parse, I think I've nailed most of them, if you find anything wrong go ahead and tell me.

Description length is either first DOT or 128 max characters (with ellipsis)
Descriptions that has "#define" in them are ignored, dunno what's up with those. Some are even FSM files, e.g https://community.bistudio.com/wiki/BIS_fnc_basicTask

Here's a pastebin with all descriptions, in a more readable format
http://pastebin.com/57A8aETf

I've also chosen to ignore BIS_fnc_module*\* and BIS_fnc_VRCourse*\* as these aren't meant to be used. Also added a check for duplicate entries (e.g BIS_fnc_customGPSVideo)

Note that running this will do ~700 HTTP GET to the BIKI, don't blame me if they block you :)

The request for function descriptions will retry 10 times, if a single one fails, no output is written.
I've also added a JSON file to easily hardcode some function descriptions.

### CODE REVIEW

**Comment by Cuel** [comment]
@jonpas: added dot at the end of descriptions and some comments

**Comment by jonpas** [comment]
Found an error in `BIS_fnc_MP`.

> this function was obsoleted since 1.

It stops on the first dot as it should, I'd say it's one of the edge cases, let me know if you want to work on it or just let it be.

Another thing I noticed when using function descriptions, for example `BIS_fnc_initVirtualUnit`'s description is:

> Initialize virtual unit Disable it, turn the camera upwards and save its correct side

There is a period after the first part of the sentence on [BI Wiki](https://community.bistudio.com/wiki/BIS_fnc_initVirtualUnit). Would be nice if a period was added automatically on newline characters.

**Comment by Cuel** [comment]
Made better support for finding dot, fixes BIS_fnc_MP and also madedirIndicator and setCuratorVIsionModes better. Hardcoded virtualnit and deprecated bis_fnc_param
